2022-01-02 23:09:05,643 INFO    MainThread:49505 [internal.py:wandb_internal():87] W&B internal server running at pid: 49505, started at: 2022-01-02 23:09:05.643242
2022-01-02 23:09:05,645 INFO    WriterThread:49505 [datastore.py:open_for_write():77] open: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/run-2tle7vrb.wandb
2022-01-02 23:09:05,645 DEBUG   HandlerThread:49505 [handler.py:handle_request():130] handle_request: check_version
2022-01-02 23:09:05,646 DEBUG   SenderThread:49505 [sender.py:send():238] send: header
2022-01-02 23:09:05,646 DEBUG   SenderThread:49505 [sender.py:send_request():252] send_request: check_version
2022-01-02 23:09:08,663 DEBUG   SenderThread:49505 [sender.py:send():238] send: run
2022-01-02 23:09:27,094 INFO    SenderThread:49505 [dir_watcher.py:__init__():169] watching files in: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files
2022-01-02 23:09:27,095 INFO    SenderThread:49505 [sender.py:_start_run_threads():825] run started: 2tle7vrb with start time 1641136144
2022-01-02 23:09:27,095 DEBUG   SenderThread:49505 [sender.py:send():238] send: summary
2022-01-02 23:09:27,095 INFO    SenderThread:49505 [sender.py:_save_file():960] saving file wandb-summary.json with policy end
2022-01-02 23:09:27,096 DEBUG   HandlerThread:49505 [handler.py:handle_request():130] handle_request: run_start
2022-01-02 23:09:27,100 DEBUG   HandlerThread:49505 [meta.py:__init__():40] meta init
2022-01-02 23:09:27,100 DEBUG   HandlerThread:49505 [meta.py:__init__():54] meta init done
2022-01-02 23:09:27,100 DEBUG   HandlerThread:49505 [meta.py:probe():214] probe
2022-01-02 23:09:27,107 DEBUG   HandlerThread:49505 [meta.py:_setup_git():204] setup git
2022-01-02 23:09:27,120 DEBUG   HandlerThread:49505 [meta.py:_setup_git():211] setup git done
2022-01-02 23:09:27,120 DEBUG   HandlerThread:49505 [meta.py:_save_pip():58] save pip
2022-01-02 23:09:27,121 DEBUG   HandlerThread:49505 [meta.py:_save_pip():72] save pip done
2022-01-02 23:09:27,121 DEBUG   HandlerThread:49505 [meta.py:_save_conda():79] save conda
2022-01-02 23:09:28,096 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_created():217] file/dir created: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/requirements.txt
2022-01-02 23:09:28,097 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_created():217] file/dir created: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/wandb-summary.json
2022-01-02 23:09:28,097 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_created():217] file/dir created: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/conda-environment.yaml
2022-01-02 23:09:33,097 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_created():217] file/dir created: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:09:35,097 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:09:35,486 WARNING FileStreamThread:49505 [file_stream.py:request_with_retry():461] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/runnphoenix/Flatland-wandb_default/2tle7vrb/file_stream (Caused by ProxyError('Cannot connect to proxy.', OSError(0, 'Error'))). func: <bound method Session.post of <requests.sessions.Session object at 0x7f15ca44cc10>>, args: ('https://api.wandb.ai/files/runnphoenix/Flatland-wandb_default/2tle7vrb/file_stream',), kwargs: {'json': {'files': {'wandb-summary.json': {'offset': 0, 'content': ['{}']}}, 'dropped': 0}}
2022-01-02 23:09:37,098 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:09:39,098 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:09:41,099 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:09:41,099 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/conda-environment.yaml
2022-01-02 23:09:41,268 DEBUG   HandlerThread:49505 [meta.py:_save_conda():89] save conda done
2022-01-02 23:09:41,268 DEBUG   HandlerThread:49505 [meta.py:probe():252] probe done
2022-01-02 23:09:41,434 DEBUG   HandlerThread:49505 [handler.py:handle_request():130] handle_request: stop_status
2022-01-02 23:09:41,434 DEBUG   SenderThread:49505 [sender.py:send_request():252] send_request: stop_status
2022-01-02 23:09:42,099 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_created():217] file/dir created: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/wandb-metadata.json
2022-01-02 23:09:46,931 DEBUG   SenderThread:49505 [sender.py:send():238] send: files
2022-01-02 23:09:46,933 INFO    SenderThread:49505 [sender.py:_save_file():960] saving file wandb-metadata.json with policy now
2022-01-02 23:09:49,101 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:09:52,100 DEBUG   HandlerThread:49505 [handler.py:handle_request():130] handle_request: stop_status
2022-01-02 23:09:52,100 DEBUG   SenderThread:49505 [sender.py:send_request():252] send_request: stop_status
2022-01-02 23:09:55,103 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:10:07,228 INFO    Thread-11 :49505 [upload_job.py:push():137] Uploaded file /tmp/tmpa1c2njhwwandb/1rhg5ncv-wandb-metadata.json
2022-01-02 23:10:08,661 ERROR   SenderThread:49505 [retry.py:__call__():126] Retry attempt failed:
Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 696, in urlopen
    self._prepare_proxy(conn)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 964, in _prepare_proxy
    conn.connect()
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connection.py", line 411, in connect
    self.sock = ssl_wrap_socket(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 428, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 472, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/home/runnphoenix/anaconda3/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/home/runnphoenix/anaconda3/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/home/runnphoenix/anaconda3/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
OSError: [Errno 0] Error

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 755, in urlopen
    retries = retries.increment(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/util/retry.py", line 573, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ProxyError('Cannot connect to proxy.', OSError(0, 'Error')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 141, in execute
    return self.client.execute(*args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/transport/requests.py", line 38, in execute
    request = requests.post(self.url, **post_args)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/api.py", line 119, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/adapters.py", line 510, in send
    raise ProxyError(e, request=request)
requests.exceptions.ProxyError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ProxyError('Cannot connect to proxy.', OSError(0, 'Error')))
2022-01-02 23:10:12,100 DEBUG   HandlerThread:49505 [handler.py:handle_request():130] handle_request: stop_status
2022-01-02 23:10:32,101 DEBUG   HandlerThread:49505 [handler.py:handle_request():130] handle_request: stop_status
2022-01-02 23:10:51,117 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:10:52,101 DEBUG   HandlerThread:49505 [handler.py:handle_request():130] handle_request: stop_status
2022-01-02 23:10:57,119 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:10:59,119 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:01,120 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:03,120 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:05,121 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:07,121 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:09,122 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:11,123 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:12,102 DEBUG   HandlerThread:49505 [handler.py:handle_request():130] handle_request: stop_status
2022-01-02 23:11:13,123 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:15,124 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:17,124 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:19,125 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:21,126 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:23,126 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:25,127 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:27,128 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:29,128 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:31,129 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:32,102 DEBUG   HandlerThread:49505 [handler.py:handle_request():130] handle_request: stop_status
2022-01-02 23:11:33,129 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:35,130 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:37,131 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:39,131 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:41,132 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:43,132 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:45,133 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:47,134 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:49,134 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:51,135 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:52,103 DEBUG   HandlerThread:49505 [handler.py:handle_request():130] handle_request: stop_status
2022-01-02 23:11:53,136 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:55,136 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:57,137 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:11:59,138 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:12:01,138 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:12:03,139 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:12:05,140 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:12:07,140 INFO    Thread-8  :49505 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:12:12,103 DEBUG   HandlerThread:49505 [handler.py:handle_request():130] handle_request: stop_status
2022-01-02 23:12:32,104 DEBUG   HandlerThread:49505 [handler.py:handle_request():130] handle_request: stop_status
2022-01-02 23:12:52,104 DEBUG   HandlerThread:49505 [handler.py:handle_request():130] handle_request: stop_status
2022-01-02 23:13:12,105 DEBUG   HandlerThread:49505 [handler.py:handle_request():130] handle_request: stop_status
2022-01-02 23:13:32,106 DEBUG   HandlerThread:49505 [handler.py:handle_request():130] handle_request: stop_status
2022-01-02 23:13:32,452 WARNING MainThread:49505 [internal.py:wandb_internal():146] Internal process interrupt: 1
2022-01-02 23:13:33,735 WARNING MainThread:49505 [internal.py:wandb_internal():146] Internal process interrupt: 2
2022-01-02 23:13:33,736 ERROR   MainThread:49505 [internal.py:wandb_internal():149] Internal process interrupted.
2022-01-02 23:13:34,460 INFO    WriterThread:49505 [datastore.py:close():281] close: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/run-2tle7vrb.wandb
2022-01-02 23:13:34,461 INFO    HandlerThread:49505 [handler.py:finish():731] shutting down handler
2022-01-02 23:15:57,385 INFO    SenderThread:49505 [sender.py:finish():1091] shutting down sender
2022-01-02 23:15:57,385 INFO    SenderThread:49505 [dir_watcher.py:finish():283] shutting down directory watcher
2022-01-02 23:15:58,173 INFO    SenderThread:49505 [dir_watcher.py:finish():313] scan: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files
2022-01-02 23:15:58,173 INFO    SenderThread:49505 [dir_watcher.py:finish():327] scan save: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/config.yaml config.yaml
2022-01-02 23:15:58,173 INFO    SenderThread:49505 [dir_watcher.py:finish():327] scan save: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/requirements.txt requirements.txt
2022-01-02 23:15:58,174 INFO    SenderThread:49505 [dir_watcher.py:finish():327] scan save: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log output.log
2022-01-02 23:15:58,177 INFO    SenderThread:49505 [dir_watcher.py:finish():327] scan save: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/wandb-summary.json wandb-summary.json
2022-01-02 23:15:58,183 INFO    SenderThread:49505 [dir_watcher.py:finish():327] scan save: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/conda-environment.yaml conda-environment.yaml
2022-01-02 23:15:58,183 INFO    SenderThread:49505 [dir_watcher.py:finish():327] scan save: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/wandb-metadata.json wandb-metadata.json
2022-01-02 23:15:58,184 INFO    SenderThread:49505 [file_pusher.py:finish():177] shutting down file pusher
2022-01-02 23:15:58,185 INFO    SenderThread:49505 [file_pusher.py:join():182] waiting for file pusher
2022-01-02 23:16:00,960 INFO    Thread-16 :49505 [upload_job.py:push():137] Uploaded file /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/conda-environment.yaml
2022-01-02 23:16:01,371 INFO    Thread-13 :49505 [upload_job.py:push():137] Uploaded file /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/requirements.txt
2022-01-02 23:16:01,408 INFO    Thread-15 :49505 [upload_job.py:push():137] Uploaded file /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/wandb-summary.json
2022-01-02 23:16:01,492 INFO    Thread-14 :49505 [upload_job.py:push():137] Uploaded file /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/output.log
2022-01-02 23:16:02,877 INFO    Thread-12 :49505 [upload_job.py:push():137] Uploaded file /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20220102_230904-2tle7vrb/files/config.yaml
2022-01-02 23:16:03,470 INFO    MainThread:49505 [internal.py:handle_exit():77] Internal process exited
